libbeat/_meta/fields.ecs.yml

```diff
@@ -2074,6 +2074,8 @@
       type: keyword
       ignore_above: 1024
       description: Hostname of the observer.
+    - name: egress.zone
+    - name: ingress.zone
```

Comment:
These fields already exist in master since the upgrade to ECS 1.5.

Reply:
@adriansr Thanks for catching that.  Should be fixed now.